spark-3.5.yaml

```diff
@@ -224,6 +245,94 @@ subpackages:
           ln -s /usr/lib/spark/sbin ${{targets.subpkgdir}}/opt/bitnami/spark/sbin
           ln -s /usr/lib/spark/yarn ${{targets.subpkgdir}}/opt/bitnami/spark/yarn
 
+  - name: ${{package.name}}-scala-2.13
+    dependencies:
+      runtime:
+        - openjdk-17-default-jvm
```

Comment:
We build with 8 and 17. We should drop this

Reply:
@EyeCantCU scala-2.13 expects to build with java-17